### ISSUE TITLE
Booking data model

### ISSUE THREAD

**Issue by albertoabellagarcia** [smart-data-models/incubated#113]
Regarding [booking data model](https://github.com/smart-data-models/incubated/tree/master/SMARTCITIES/SMARTPORTS/Booking) 
@majuse 
1) Is it possible that there would be more than 2 container export or import? 
this was the case the schema should be redefined to an array of items (where there is no limit to the elements)
2) "bookingStatus" is defined as string but the example is an array
3) There are quite a lot of mandatory attributes in the required clause. Why is that?

**Comment by majuse** [comment]
Good afternoon,

I have forwarded your questions to the port that is currently making use 
of the Booking data model (port of Thessaloniki). These are their answers:

1. The attributes for the import and export containers have been defined 
that way because 2 is the maximum number of containers that can be 
carried by a truck:

"The truck may enter the Port carrying ONE container (to drop off), and 
as it is in, it may load TWO containers and take them out of the port 
(pick-up). Or drop-off two containers, and pick-up one. Or drop-off two, 
and pick-up two. Or drop-off nothing and pick-up two. Or drop-off two 
and pick-up nothing. You get the idea, it’s not exactly /any/ number, 
it’s this combination I described."


2. It was a mistake. I have fixed the example.


3. The attributes that were defined as mandatory are always present in 
the case of the port of Thessaloniki. However, some of the attributes 
could be defined as optional considering a more general scentario: 
"company", "containersExport" and "containersImport" could be defined as 
optional, keeping only the booking number, booking date and status as 
mandatory.


Best regards,

   Matilde
